docs/src/pages/[platform]/components/grid/demo.tsx

```diff
@@ -1,17 +1,98 @@
-import React, { useState } from 'react';
+import * as React from 'react';
 import {
   Card,
+  Expander,
+  ExpanderItem,
   Grid,
   GridContainerStyleProps,
   GridItemStyleProps,
   useTheme,
-  View,
 } from '@aws-amplify/ui-react';
-import { useGridContainerProps } from '@/components/useGridContainerProps';
-import { Example } from '@/components/Example';
-import { GridContainerPropControls } from '@/components/GridContainerPropControls';
-import { GridItemPropControls } from '@/components/GridItemPropControls';
-import { useGridItemProps } from '@/components/useGridItemProps';
+
+import { Demo } from '@/components/Demo';
+import { demoState } from '@/utils/demoState';
+import { GridContainerPropControls } from './GridContainerPropControls';
+import {
+  GridItemPropControls,
+  GridItemPropControlsProps,
+} from './GridItemPropControls';
+import { useGridItemProps } from './useGridItemProps';
+import { useGridContainerProps } from './useGridContainerProps';
+
+const propsToHeaderGridItemCode = (props: GridItemPropControlsProps) => {
+  return (
+    '<Card' +
+    (props.columnStart ? `\n  columnStart={${props.columnStart}}` : '') +
```

Comment:
Just a heads up that we have a `getPropString` function in the `utils` folder that does this for us

Reply:
ohhh, I like this util.